test/navigationLibrary/StandardRoute/RouteSegmentCollection.spec.js

```diff
@@ -53,13 +54,21 @@ ava('.findWaypointsForSegmentName() returns an array of fixes for a given segmen
     t.true(_isEqual(result[0], ['PIRMD', null]), 'Expected to receive and array that is [`{string}`, `{string|null}`]');
 });
 
-ava('.gatherFixNamesForCollection() returns a list of names for each RouteSegmentModel in the collection', t => {
+ava('.gatherFixNamesForCollection() returns a list of entryPoint/exitPoint names for each RouteSegmentModel in the collection', t => {
     const collection = new RouteSegmentCollection(ROUTE_SEGMENTS_MOCK);
     const result = collection.gatherFixNamesForCollection();
 
     t.true(result.length === 8);
 });
 
+ava('.getAllFixNames() returns an array of fix names used by any segment in the collection', (t) => {
+    const collection = new RouteSegmentCollection(ROUTE_SEGMENTS_MOCK);
+    const result = _uniq(collection.getAllFixNames().sort());
```

Comment:
After thinking about it more, the function itself should be returning a unique array anyway, so amended it as such, and removed the uniq from the test, while leaving the unique `expectedResults` (effectively verifying it the same way).

done